styles/styles.css

```
@@ -0,0 +1,553 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("/fonts/ptsans-400.woff2") format("woff2");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("/fonts/ptsans-700.woff2") format("woff2");
+  font-display: swap;
+}
+
+body {
+  font-family: "PT Sans", sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+  background-color: #FFFFFF;
+  color: #333333;
+}
+
+.header-button {
+  width: 160px;
+  height: 37px;
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 20px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #ffffff;
+  background: #756257;
+  border-radius: 4px;
+  text-decoration: none;
+
+}
+
+.btn-search {
+  width: 575px;
+  height: 60px;
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 36px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #ffffff;
+  background: #756257;
+  border-radius: 10px;
+  text-decoration: none;
+
+}
+
+.button-distribution {
+  width: 200px;
+  height: 61px;
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 26px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #ffffff;
+  background: #83B3D3;
+  border-radius: 0px 10px 10px 0px;
+  text-decoration: none;
+
+}
+
+.distribution-write {
+  width: 600px;
+  height: 60px;
+  background: #F2F2F2;
+  border-radius: 0px 10px 10px 0px;
+}
+
+.button-accept {
+  width: 160px;
+  height: 48px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 21px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #ffffff;
+  background: #83B3D3;
+  border-radius: 4px;
+  text-decoration: none;
+}
+
+.button-reset {
+  width: 160px;
+  height: 23px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 23px;
+  text-align: center;
+  color: #FFFFFF;
+  background-color: rgba(0, 0, 0, 0);
+  border: rgba(0, 0, 0, 0)
+}
+
+.product-card-button {
+  /*кажется что с ссылкой/кнопкой что то не так визуально, не пойму в чем дело*/
+  width: 133px;
+  height: 37px;
+  background: #756257;
+  border-radius: 4px;
+  font-family: PT Sans;
```

```suggestion
 font-family: "PT Sans", sans-serif;
```

https://up.htmlacademy.ru/htmlcss/33/criteries#CSS-06

---

везде поправить

---

пока не сделано до конца